index.html

```diff
@@ -1,16 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
     <title>Koombea Website</title>
-</head>
-<h1>Koombea</h1>
-
-<body>
-
-</body>
+  </head>
 
-</html>
\ No newline at end of file
+  <body>
+    <header id="header" class="header">
+      <div class="container">
+        <input type="checkbox" id="check" class="check" />
+        <nav class="navbar navbar-top">
+          <label for="check" class="checkbtn">
+            <img src="img/logo/menu.svg" alt="logo hamburguer menu" />
+          </label>
+          <a href="#" class="logo"
+            ><img src="img/logo/logo-koombea.svg" alt="logo koombea"
+          /></a>
+          <a href="#" class="logo-responsive"
+            ><img
+              src="img/logo/logo-responsive.svg"
+              alt="logo koombea responsive"
+          /></a>
+          <ul class="navbar-list">
+            <li class="list"><a href="#" class="list-link current">Home</a></li>
```

Comment:
This element should have this class when is active: `list-link--active` @Jefer-Max